syntax = "proto3";

service Sorter {
    rpc sort (sortRequest) returns (sortReply); // orders elements
}


message sortRequest{
    string sortReq = 1;
}
message sortReply{
    string sortRep = 1;
}

//COMMAND LINE TO COMPILE: python -m grpc_tools.protoc -I ./ --python_out=. --grpc_python_out=. ./sorter.proto